client/pages/sign-up/sign-up.js

```diff
@@ -1,21 +1,20 @@
-import HeaderComponent from "../../components/header-component.js";
+// import HeaderComponent from "../../components/header-component.js";
 // import FooterComponent from "../../components/footer-component.js";
 import MainSection from "./main-section.js";
 import FormSection from "./form-section.js";
 
 const SignUpPage = Vue.createApp({
    components: {
-      HeaderComponent,
+      // HeaderComponent,
       // FooterComponent,
       MainSection,
       FormSection,
    },
    template: `
-      <div class="page">
+      <div class="sign-page">

```

Comment:
Please include the header-component and footer-component

Reply:
I included them. But I forgot to uncomment this part. I will fix it!